ab_tool/models.py

```diff
@@ -299,6 +304,12 @@ def get_emails(self):
     
     def notification_sent(self):
         self.update(last_emailed=timezone.now())
+    
+    def get_canvas_domain(self):
```

Comment:
The method name here is a bit misleading.  Something with the name `course` would be better.